.github/workflows/create-update-issues.yaml

```diff
@@ -0,0 +1,80 @@
+name: Create Update Issues
+
+on:
+  workflow_call:
+    inputs:
+      AUTH_APP_ID:
+        description: Github app id to authenticate in mobile and extention repositories
+        type: string
+        required: true
+    secrets:
+      AUTH_APP_PRIVATE_KEY:
+        description: Github app private key to authenticate in mobile and extention repositories
+        required: true
+
+jobs:
+  create-update-issues:
+    permissions:
+      issues: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout head
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get release tag
+        id: releaseTag
+        run: |
+          echo "tag=$(git describe --tags --abbrev=0)" >> "$GITHUB_OUTPUT"
```

Comment:
Could `publish-release` step output created tags as it have it? This would simplify whole process. We can use also use `git tag --points-at head` command and process output it the loop

Reply:
I don't think that would be easily accomplished in this case, because of how the publish workflow/action is setup.

However, getting all tags on the HEAD commit is quite easy: `git describe --tags --match '@metamask/*' HEAD`